src/BizHawk.Emulation.Common/Base Implementations/MemoryDomain.cs

```diff
@@ -108,7 +108,7 @@ public virtual void BulkPeekByte(Range<long> addresses, byte[] values)
 			if (addresses is null) throw new ArgumentNullException(paramName: nameof(addresses));
 			if (values is null) throw new ArgumentNullException(paramName: nameof(values));
 
-			if ((long) addresses.Count() != values.Length)
+			if ((long) addresses.Count() > values.Length)

```

Comment:
Why is this necessary? And is it part of the BANK register changes?

Reply:
I was having issues with GBHawk requiring memory mapped buffer to match a 0x10000 size, so it was excepting incorrectly. It is unrelated to banking code, I'll revert that part and fix separately as part of a separate change.

Reply:
@YoshiRulz , finally got back to this, went ahead and reverted this particular change. I need to resync my branch and make sure nothing got needs to be updated in the few months since my last commit (other then probably rebasing my gambette branch, or working with you to get my code merge into yours over there.)